### DISCUSSION TITLE
Chia Plots not recognised by Chia app

### DISCUSSION THREAD

**Discussion by MadKing1** [Chia-Network/chia-blockchain#6730] [Support]
Hi, today my PC was rebooted and after that all plots are gone from app (they are on drives, all is ok with files). But Chia cannot find them. I had add location of them again and again, later uninstall chia app - nothing helps.

After adding again it shows there is no plots but on a bottom list they are mentioned... Any help what to do? Im noobie so please try to say it in common language if possible :D 

thank you very much!!!
![image](https://user-images.githubusercontent.com/85879484/121889132-22ffd480-cd19-11eb-8e51-a38a202a8e68.png)

**Comment by loppefaaret** [comment]
COMPLETE CONFIG RESET

Close down the Chia app, and open up a PowerShell window, and execute the below commands one at a time:
```
mv ~\.chia\mainnet ~\.chia\old
cd ~\AppData\Local\chia-blockchain\app-1.1.7\resources\app.asar.unpacked\daemon\
.\chia.exe init
mv ~\.chia\old\db ~\.chia\mainnet\
mv ~\.chia\old\wallet ~\.chia\mainnet\
```
after this, you can open up the Chia app again, and readd you plots again, and it should now work - remember to redo any settings you had set before the reset!

**Comment by BrandtH22** [comment]
@MadKing1 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
